paasta_tools/utils.py

```diff
@@ -841,7 +842,7 @@ def _now():
 
 def remove_ansi_escape_sequences(line):
     """Removes ansi escape sequences from the given line."""
-    return no_escape.sub('', line)
+    return no_escape.sub('', line) if type(line) is str else no_escape_b.sub(b'', line)
```

Comment:
It breaks ~30 tests.  I'm fine with updating them to all use bytes if that's what we want.  I just didn't know the context in which this is used so I opted for the most simple change until I got more feedback.